query string in access application paths (#22006)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/policies/access/app-paths.mdx

Port numbers are not supported in Access application paths. If a request includes a port number in the URL, Access will strip the port number and redirect the request to the default HTTP/HTTPS port.

### Query strings

> Query strings (e.g. `?foo=bar`) are not supported in Access application paths.

Reviewer: ```suggestion
Query strings (such as`?foo=bar`) are not supported in Access application paths.
```
